### ISSUE TITLE
Snap store seems to be installing x86_64 version on arm64 hardware

### ISSUE THREAD

**Issue by Cuperino** [Cuperino/QPrompt#165]
Snap store seems to be installing x86_64 version on arm64 hardware. This seems to have been the case on @videosmith's Ubuntu 22.10 Raspberry Pi install.

**Comment by Cuperino** [comment]
This issue should be fixed with v1.1.6 Snap release. Needs testing before issue can be closed.

**Comment by videosmith** [comment]
Will 1.1.6 have additions/fixes vs 1.2.0?
    On Sunday, April 9, 2023 at 02:18:11 PM EDT, Javier O. Cordero Pérez ***@***.***> wrote:  
 
 


This issue should be fixed with v1.1.6 Snap release. Needs testing before issue can be closed.

**Comment by Cuperino** [comment]
No. Aside from changes to the [script for creating Snap builds](https://github.com/Cuperino/qprompt-snap), the only difference between v1.1.5 and v1.1.6 is the fix to bug #160, which has been fixed in the v1.2.0 branch since [Early Access QPrompt v1.2.0 build 03](https://www.patreon.com/posts/early-access-v1-79849057).
